README.rst

```diff
@@ -22,7 +22,7 @@ of a component, check the `Home Assistant help section <https://home-assistant.i
 
 .. |Chat Status| image:: https://img.shields.io/discord/330944238910963714.svg
    :target: https://www.home-assistant.io/join-chat/
-.. |screenshot-states| image:: https://raw.githubusercontent.com/home-assistant/core/master/docs/screenshots.png
+.. |screenshot-states| image:: https://raw.githubusercontent.com/xMohamd/home-assistant-core/README-image-fix/.github/assets/screenshot-states.png
```

Comment:
We shouldn't link to your branch in your fork. We should link to the dev branch in this upstream repository.

Reply:
As I understand it I need to change links, as in the example below right?
```link
https://raw.githubusercontent.com/home-assistant/core/dev/.github/assets/screenshot-states.png
```